docs/src/examples/components/Segment/Variations/SegmentExampleColor.shorthand.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import _ from 'lodash'
+import { Segment, ProviderConsumer } from '@stardust-ui/react'
+
+const SegmentExampleColor = () => (
+  <ProviderConsumer
+    render={({ siteVariables: { emphasisColors, naturalColors } }) =>
+      _.keys({ ...emphasisColors, ...naturalColors }).map(color => (
+        <>
+          <Segment key={color} color={color} content={_.startCase(color)} inverted />
+          <br />
+        </>
```

Comment:
For simplicity suggest turning:
```jsx
         <>
           <Segment key={color} color={color} content={_.startCase(color)} inverted />
           <br />
         </>
```
into:
```jsx
<Segment key={color} color={color} content={color} inverted />
```

Reply:
simplified, thanks